jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/GroovyMetricEnvironment.java

```diff
@@ -139,7 +178,16 @@ protected static Attributes mapToAttributes(@Nullable final Map<String, String>
    */
   public DoubleCounter getDoubleCounter(
       final String name, final String description, final String unit) {
-    return meter.counterBuilder(name).setDescription(description).setUnit(unit).ofDoubles().build();
+    AtomicReference<DoubleCounter> existingCounter =
+        doubleCounterRegistry.getExisting(
+            name, description, unit, InstrumentType.COUNTER, InstrumentValueType.DOUBLE);
+
+    DoubleCounter dc = existingCounter.get();
+    if (dc == null) {
+      dc = meter.counterBuilder(name).setDescription(description).setUnit(unit).ofDoubles().build();
+      existingCounter.set(dc);
+    }
+    return dc;
```

Comment:
I don't know the jmx-metrics code well, but these registries should not be necessary: if you ask a meter for an instrument multiple times with the same name, description, unit, type, and value type, there is no conflict and you won't get any warning log. 

Warnings are only produced if you request an instrument with the same name as one previous, but with a different description, unit, type, or value type. 

This code only seems to ensure that the same instrument is returned if it is requested with the same name, description, unit, type, and value type.